### PR TITLE
Claire/navigation

### PR DESCRIPTION
**Pull request by clairelin135**
Adds a navbar.
- By default, displays the `works#index` page
- Routes to a listing of all artists
- Displays links to artist login and patron login if nobody is logged in. Otherwise, displays logout button and profile button for logged-in user.

Adds an `AllArtists` screen.
- Index of all artist names
- Links to profile pages of each artist via the `artists#show` method

Adds a `BuyerProfile` screen.
- Nothing on it! It will just be necessary for future PRs.

Stylistic Additions
- Provides some general guidelines for `.scss` styling, by fixing scss display
- Adds some general colors and styles that are meant to be changed in the future!

### Related PRs
Barbara's `AllWorks` page could have merge conflicts, but should be pretty rudimentary to resolve.

### Migrations
Creates a `Buyers` controller. Adds view for buyer profile.

### Screenshots
image

CC: @clairelin135

### CODE REVIEW

**Comment by clairelin135** [comment]
Overwrite styles later!

File: app/controllers/artists_controller.rb [hunk 1]
```diff
@@ -6,4 +6,7 @@ def index
   def show
     @artist = Artist.find(params[:id])
   end
+
+  def all_artists
```
Comment (calvind777) [inline app/controllers/artists_controller.rb]: what's the reason for this method?

Comment (clairelin135) [reply app/controllers/artists_controller.rb]: It serves as an index for listing all artists. This page will later be modified to serve as this page: 
image

File: app/assets/stylesheets/partials/_reset.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  content: none;
+}
+
+a img {
+  border: none;
+}
+
+article, aside, details, figcaption, figure, footer, header, hgroup, main, menu, nav, section, summary {
+  display: block;
+}
```
[the remark this answers is not in the document]
Comment (clairelin135) [reply app/assets/stylesheets/partials/_reset.scss]: Liang's SJC reset css file :P

**Comment by calvind777** [comment]
update

**Comment by calvind777** [comment]
update

**Pull request merged**